service-deployment/clusterconfig-example.yaml

```diff
@@ -44,6 +44,8 @@ clusterinfo:
     docker_username: your_registry_username
     docker_password: your_registry_password
 
+    docker_tag: your_image_tag
```

Comment:
After we finished initial deployment and want to upgrade cluster images to a newer tag, what's the procedures to do it?

Reply:
@sterowang 
1) Rolling-update: https://kubernetes.io/docs/tasks/manage-daemon/update-daemon-set/
2) Rolling-back: https://kubernetes.io/docs/tasks/manage-daemon/rollback-daemon-set/

Currently we can't directly enable the upgrade feature. Before it, we should add the option ``` .spec.updateStrategy.type = RollingUpdate.```. Then we have two methods to do it.

a: Directly use kubectl
b: Develop a tool to do upgrading and package the function.